tests/boardfarm_plugins/boardfarm_prplmesh/prplmesh_config.json

```diff
@@ -1,19 +1,25 @@
 {
     "prplmesh_docker": {
-        "name": "dockerized_device",
+        "name": "controller",
         "board_type": "prplmesh_docker",
         "role": "controller",
         "conn_cmd": "",
         "devices": [
             {
-                "name": "repeater1",
+                "name": "lan",
```

Comment:
Why rename this? And as usual:
- should have been a separate commit;
- should be explained why in the commit message.

(You don't need to split it in a separate commit now, but please do explain in the commit message.)

If the rename is not needed, I'd much prefer it.

Reply:
Without rename, we can't address devices by name.